pyDeltaRCM/sed_tools.py

```diff
@@ -330,14 +379,24 @@ def _compute_Vp_ero(self, Vp_sed, U_loc, U_ero, beta):
         return (Vp_sed * (U_loc**beta - U_ero**beta) /
                 U_ero**beta)
 
-    def _limit_Vp_change(self, Vp, stage, eta, dx):
+    def _limit_Vp_change(self, Vp, stage, eta, dx, dep_ero):
         """Limit change in volume to 1/4 of a cell volume.
 
         Function is used by multiple pathways in `mud_dep_ero` and `sand_dep_ero`
         but with different inputs for the sediment volume (:obj:`Vp`).
+
+        dep_ero indicates which pathway we are in, dep==0, ero==1
```

Comment:
Maybe clarify in this doc-string that the volume returned is always going to be positive (even in the case of erosion)?

Or, change the function to check `dep_ero` and return `-1*np.minimum(Vp, fourth)` in the erosion case. This might clean up future calls in erosion scenarios where this function call is always followed by `Vp_change = Vp_change * -1` - although a clean solution for this that doesn't add another `if-else` check is not obvious to me...

Reply:
good idea, I've update this docstring and another to make clear that the values are positive, and making them negative should be handled in the Router-specific implementation of `_deposit_or_erode`.